integration_tests/test_staking_precompile.py

```
@@ -0,0 +1,265 @@
+    assert balance == balance_bf - sum(amounts) * WEI_PER_UOM - fee
+
+    unbonded_bf = cli.staking_pool(bonded=False)
+    unbonded_amt = 2
+    res = await UNDELEGATE(acct.address, val_ops[i], unbonded_amt).transact(
+        w3, acct.address, to=STAKING
+    )
+    assert res.status == 1
+    fee += res["gasUsed"] * res["effectiveGasPrice"]
+    assert cli.staking_pool(bonded=False) == unbonded_bf + unbonded_amt
+    blk = res["blockNumber"]
+    rsp = requests.get(f"{cli.node_rpc_http}/block_results?height={blk}").json()
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)